src/drive/mobile/containers/settings/About.jsx

```diff
@@ -1,26 +1,48 @@
-import React from 'react'
+import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import { translate } from 'cozy-ui/react/I18n'
 import SettingCategory, { ELEMENT_TEXT } from '../../components/SettingCategory'
+import Hammer from 'hammerjs'
 
-export const About = ({ t, version, serverUrl, onClick }) => (
-  <SettingCategory
-    title={t('mobile.settings.about.title')}
-    elements={[
-      {
-        type: ELEMENT_TEXT,
-        label: t('mobile.settings.about.account'),
-        value: <a href="{serverUrl}">{serverUrl}</a>
-      },
-      {
-        type: ELEMENT_TEXT,
-        label: t('mobile.settings.about.app_version'),
-        value: version,
-        onClick
-      }
-    ]}
-  />
-)
+class About extends Component {
+  componentDidMount() {
+    this.gesturesHandler = new Hammer.Manager(this.versionNumber)
+    this.gesturesHandler.add(new Hammer.Tap({ event: 'singletap' }))
+    this.gesturesHandler.on('singletap', this.props.onTap)
```

Comment:
Il n'y a pas un moyen d'enregistrer directement un event `{ event: 'tripletap' }` ?

Reply:
It… could make sense I guess but I don't know how to do that. @y-lohse what do you think?

Reply:
Something like this should to the trick : 

```
this.gesturesHandler.add(new Hammer.Tap({ event: 'tripletap', taps: 3 }))
this.gesturesHandler.on('tripletap', this.props.onTap)
```

Haven't tried it but I guess it would work.

Reply:
My thoughts are: if we use a third-party library to handle _tap_ and/or _touch_ events, it could be more stable to deport the responsibility to the library. But if there is no built-in support for _triple tap_, then we can handle the tap sequence in our code.